recipe/meta.yaml

```diff
@@ -325,7 +331,7 @@ outputs:
       # build cuQuantum Python package is CUDA agnostic, serving both CUDA 11/12
       # only build in the CUDA 11 pipeline
       # (note: this still uses cuda_compiler_version to calculate hash)
-      skip: true  # [not (cuda_compiler_version or "").startswith("11")]
+      skip: true  # [not (cuda_compiler_version or "").startswith("12")]
```

Comment:
so now we only build in the CUDA 12 pipeline? nits: line 332 should change the documentation

Reply:
Right. We used to only build in the cuda 11 pipeline, now we only build in the cuda 12 pipeline. Motivation for this is:
- CUDA12 pipeline should be slightly faster due
- CUDA11 support will be dropped at some point in the future.

Reply:
I will apply the comment change with the next commit, otherwise I'll postpone it to a future releases' update of the recipe.